### PR TITLE
Enable metadata tests

### PR DESCRIPTION
**Pull request by berquist**
as mentioned in https://github.com/cclib/cclib/pull/732#issuecomment-530198350.

I also found a few other skip decorators in the single-point tests that could be removed.

### CODE REVIEW

File: test/data/testSP.py [hunk 1]
```diff
@@ -293,20 +290,11 @@ def testmoments(self):
...
-    @skipForParser('GAMESSUK', 'Does not support metadata yet')
-    @skipForParser('Molcas', 'The parser is still being developed so we skip this test')
-    @skipForParser('Molpro', 'Does not support metadata yet')
-    @skipForParser('NWChem', 'Does not support metadata yet')
-    @skipForParser('ORCA', 'Does not support metadata yet')
-    @skipForParser('Psi3', 'Does not support metadata yet')
-    @skipForParser('Psi4', 'Does not support metadata yet')
-    @skipForParser('QChem', 'Does not support metadata yet')
-    @skipForParser('Turbomole', 'The parser is still being developed so we skip this test')
     def testmetadata(self):
```
Comment (langner) [inline test/data/testSP.py]: Do you think it makes sense to break this down into one method for each metadata attribute? Do we want to validate the value of any of the metadata?

Comment (berquist) [reply test/data/testSP.py]: > Do you think it makes sense to break this down into one method for each metadata attribute?

Yes.

> Do we want to validate the value of any of the metadata?

Yes, but related to https://github.com/cclib/cclib/pull/599#pullrequestreview-149546363 and https://github.com/cclib/cclib/issues/602, testing parser-specific results is messy right now, unless we restrict it to `SP` tests.

Comment (langner) [reply test/data/testSP.py]: I think it's fine to limit to `SP` test for now in that case. Maybe sprinkle with TODOs so we now where to go from here. Anyway, we can do this incrementally.

**Comment by langner** [comment]
@berquist is this ready for v1.6.3?

**Comment by berquist** [comment]
@langner Yes.

**Comment by langner** [comment]
OK, here we go!

**Pull request merged**